modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc

```diff
@@ -37,10 +37,11 @@ where:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
--e DISCOVERY_MODE=true -e FEATURES=performance -e IMAGE_REGISTRY="<disconnected_registry>" \
+podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
+-e IMAGE_REGISTRY="<disconnected_registry>" \
 -e CNF_TESTS_IMAGE="cnf-tests-rhel8:v{product-version}" \
-/usr/bin/test-run.sh -ginkgo.focus="\[performance\]\ Latency\ Test"
+-e LATENCY_TEST_RUNTIME=<time in seconds> \
+<disconnected_registry>/cnf-tests-rhel8:v{product-version} /usr/bin/test-run.sh --ginkgo.v
 ----
 
 [discrete]
```

Comment:
Subheadings in procedure modules are not allowed by mod docs guidelines. This is outside the scope of this PR, so just a note, no need to fix right now.